proto/storage/image.proto

```diff
@@ -79,6 +83,12 @@ message ImageScan {
 
 message ImageSignatureVerificationData {
     repeated ImageSignatureVerificationResult results = 1;
+    // The full image names that are verified (meaning, having a successful signature verification) keyed by the
```

Comment:
I think the comment makes the most sense above the `verified_image_references` field.